cmd/ocm-backplane/cloud/common.go

```diff
@@ -284,10 +295,34 @@ func (cfg *QueryConfig) getIsolatedCredentials(ocmToken string) (aws.Credentials
 		return aws.Credentials{}, fmt.Errorf("failed to assume role using JWT: %w", err)
 	}
 	// Verify Sts connection with the seed credentials
-	stsClient := sts.NewFromConfig(aws.Config{
-		Region:      "us-east-1",
-		Credentials: NewStaticCredentialsProvider(seedCredentials.AccessKeyID, seedCredentials.SecretAccessKey, seedCredentials.SessionToken),
-	})
+	// Use AWS-specific proxy for STS operations
+	var awsProxyURL *string
+	if awsProxy := os.Getenv("BACKPLANE_AWS_PROXY"); awsProxy != "" {
```

Comment:
Seems like a repetition of the same logic. This may be avoided if you move the logic to BackplaneConfig function